### PR TITLE
8245499: Text input controls should show handles on iOS

### PR DESCRIPTION
**Pull request by jperedadnr**
After JDK-8242167, a JavaFX control is used for text input on iOS instead of the native control, on a touch enabled device. However, selection handles are not enabled and currently text selection is not possible at all. 

This PR enables handles on iOS as in the rest of the platforms with touch support.

It also tries to accommodate its style to the native style for handles, where selection handles have a circle shape and are a little bit bigger, and there is no caret handle.

I've tested it on iPhone, running a small [HelloFX](https://github.com/gluonhq/client-samples/tree/master/Maven/HelloFX) sample, with a TextField control.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8245499](https://bugs.openjdk.java.net/browse/JDK-8245499): Text input controls should show handles on iOS

### Reviewers
 * Ajit Ghaisas ([aghaisas](@aghaisas) - **Reviewer**)
 * Johan Vos ([jvos](@johanvos) - **Reviewer**)

### Download
`$ git fetch https://git.openjdk.java.net/jfx pull/231/head:pull/231`
`$ git checkout pull/231`

### CODE REVIEW

**Comment by kevinrushforth** [comment]
Since this touches shared code, I'd like @aghaisas to look at this.

File: modules/javafx.controls/src/ios/resources/com/sun/javafx/scene/control/skin/caspian/ios.css [hunk 1]
```diff
@@ -51,3 +51,8 @@
     -fx-skin: "javafx.scene.control.skin.TextAreaSkinIos";
 }
 
+.selection-handle {
+    -fx-shape: "M100,100m-75,0a75,75,0,1,0,150,0a75,75,0,1,0,-150,0";
+    -fx-pref-width: 12;
+    -fx-pref-height: 12;
+}
```
Comment (aghaisas) [inline modules/javafx.controls/src/ios/resources/com/sun/javafx/scene/control/skin/caspian/ios.css:58]: No newline at end of file.

File: modules/javafx.controls/src/ios/resources/com/sun/javafx/scene/control/skin/modena/ios.css [hunk 2]
```diff
@@ -51,3 +51,8 @@
     -fx-skin: "javafx.scene.control.skin.TextAreaSkinIos";
 }
 
+.selection-handle {
+    -fx-shape: "M100,100m-75,0a75,75,0,1,0,150,0a75,75,0,1,0,-150,0";
+    -fx-pref-width: 12;
+    -fx-pref-height: 12;
+}
```
Comment (aghaisas) [inline modules/javafx.controls/src/ios/resources/com/sun/javafx/scene/control/skin/modena/ios.css:58]: No newline at end of file.

**Comment by aghaisas** [comment]
Code changes look OK.
I think, we need a note in PR description about what testing you have done with this fix.

File: modules/javafx.controls/src/ios/resources/com/sun/javafx/scene/control/skin/caspian/ios.css (hunk 1 above)
Comment (jperedadnr) [reply modules/javafx.controls/src/ios/resources/com/sun/javafx/scene/control/skin/caspian/ios.css]: Fixed

File: modules/javafx.controls/src/ios/resources/com/sun/javafx/scene/control/skin/modena/ios.css (hunk 2 above)
Comment (jperedadnr) [reply modules/javafx.controls/src/ios/resources/com/sun/javafx/scene/control/skin/modena/ios.css]: Fixed

**Comment by jperedadnr** [comment]
I've edited the PR description with details of the tests I've done.

**Pull request closed without merging**